### PR TITLE
Fixed the fields display issue when they are assigned into fieldGroups

### PR DESCRIPTION
**Pull request by yzhao583**
Fixes:

https://jira.coreos.com/browse/CONSOLE-1890

### CODE REVIEW

**Comment by tlwu2013** [comment]
Hey @spadgett / @alimobrem , can you help review this PR?  This PR fixed the fieldGroup rendering issue captured for 4.3 Console in Console 1890: 
_"fieldGroup" - fields being included into `fieldGroup` always get displayed as "text label" (default display) on Operand Details View_
https://jira.coreos.com/browse/CONSOLE-1890

Much appreciated!

File: frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx [hunk 1]
```diff
@@ -712,10 +712,12 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
               return (
                 <div id={group} key={group}>
                   <FieldGroup
-                    defaultExpand={_.some(
-                      fieldList,
-                      (f) => f.capabilities.includes(SpecCapability.advanced) && !f.required,
-                    )}
+                    defaultExpand={
+                      !_.some(
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx]:
```suggestion
                      _.every(
```

File: frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx [hunk 2]
```diff
@@ -754,10 +756,12 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
...
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx]:
```suggestion
                      _.every(
```

File: frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/index.tsx [hunk 3]
```diff
@@ -106,8 +107,48 @@ const BooleanSwitch: React.FC<SpecCapabilityProps> = (props) => {
...
+            Confirm change
+          </Button>
+        </>
+      )}
+    </div>
+  );
+};
+
+const CheckboxUIComponent: React.FC<SpecCapabilityProps> = (props) => {
+  const convertedValue = !_.isNil(props.value) ? props.value : false;
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/index.tsx]:
```suggestion
  const convertedValue: boolean = !_.isNil(props.value) ? props.value : false;
```

**Pull request merged**